packages/perseus/src/widgets/__tests__/expression.test.ts

```diff
@@ -15,47 +15,64 @@ import {Expression} from "../expression";
 
 import {renderQuestion} from "./renderQuestion";
 
-type Item = any;
-
-const assertComplete = (itemData: Item, input, isCorrect: boolean) => {
+const assertComplete = (itemData: PerseusItem, input, isCorrect: boolean) => {
     const {renderer} = renderQuestion(itemData.question);
     userEvent.type(screen.getByRole("textbox"), input);
     const [_, score] = renderer.guessAndScore();
+
     expect(score).toMatchObject({
         type: "points",
         earned: isCorrect ? 1 : 0,
     });
+
+    expect(testDependencies.analytics.sendEvent).toHaveBeenCalledWith({
+        eventType: "ContentLibraryMathInputBoxEvaluated",
+        eventSchemaVersion: 2,
+        virtualKeypadVersion: "PERSEUS_MATH_INPUT",
+        evaluationResult:
+            score.type === "invalid"
+                ? "invalid"
+                : score.earned === 1
+                ? "correct"
+                : "incorrect",
+    });
```

Comment:
I'm kind of leery about these kinds of assertions in tests because it includes logic that also has to be right.  I feel like we should be moving in the opposite direction where we use `expect` directly in the test case and pass the exact object literal we want to compare against instead of ones containing logic like this.